packages/toml/src/index.ts

```diff
@@ -0,0 +1,59 @@
+import { Taplo } from '@taplo/lib'
+import type { Plugin } from 'prettier'
+
+import { languages } from './languages.js'
+import { prettierOptionsDefinitions } from './options.js'
+import type { PrettierOptions, TaploOptions } from './types.js'
+
+const PLUGIN_NAME = 'toml'
+
+let taplo: Taplo | null = null
```

Comment:
```suggestion
let taplo: Taplo
```

I'd prefer no wasting initialization.